include/func.sh

```diff
@@ -377,10 +397,22 @@ wait_for_cf-operator() {
       # qstate_tolerations fails when internet connectivity is disabled.
       wait_for "kubectl apply -f ../kube/cf-operator/qstatefulset_tolerations.yaml --namespace=scf"
     fi
-    wait_ns scf
+    wait_for_scf
     #wait_for "kubectl delete -f ../kube/cf-operator/boshdeployment.yaml --namespace=scf"
     wait_for "kubectl delete -f ../kube/cf-operator/password.yaml --namespace=scf"
     if [[ "${DOCKER_REGISTRY}" == "registry.suse.com" ]]; then
       wait_for "kubectl delete -f ../kube/cf-operator/qstatefulset_tolerations.yaml --namespace=scf"
     fi
 }
+
+wait_for_scf() {
```

Comment:
If I remember correctly, `scf` doesn't have operator, just `kubecf` right? And I see that we didn't update `modules/scf/install.sh` to use the new code (probably because it doesn't have `ig` jobs).  Shouldn't this method be called `wait_for_kubecf` instead, then?

Reply:
Yes, I guess. It is `wait_ns scf` but actually waits for `kubecf`, which is totally not confusing or anything...